#!/bin/sh /usr/share/dpatch/dpatch-run

## Makefile-fPIC-fix patch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adds -fPIC to Makefiles to fix potential FTBFS.

@DPATCH@
diff -urNad /usr/src/cvs/vdrdevel-plugin-streamdev/Makefile vdrdevel-plugin-streamdev/Makefile
--- /usr/src/cvs/vdrdevel-plugin-streamdev/Makefile	Mon Feb 16 13:08:37 2004
+++ vdrdevel-plugin-streamdev/Makefile	Sat Sep 11 15:29:36 2004
@@ -16,7 +16,7 @@
 ### The C++ compiler and options:
 
 CXX      ?= g++
-CXXFLAGS ?= -W -Woverloaded-virtual
+CXXFLAGS ?= -fPIC -W -Woverloaded-virtual
 
 ### The directory environment:
 
